cfgov/ask_cfpb/jinja2/ask-cfpb/answer-page.html

```diff
@@ -22,8 +22,8 @@
 
```

Comment:
The viz checker is showing that Ask pages are a little broken. I believe changing `ask-cfpb-page__answer` to `ask-cfpb-page--answer` on L20 in this file fixes things.

Reply:
Fixed in https://github.com/cfpb/consumerfinance.gov/pull/8274